operations/workplace/people/README.md

```diff
@@ -1,15 +1,15 @@
 ---
-description: >-
+Description: >-
   Welcome to the Mattermost People Team Handbook Page. This page is being
   updated and actively worked on. If you have any question that you don't see
-  answered here, see PeopleOps on Mattermost.
+  answered here, see People Ops on Mattermost.
 ---
 
-# PeopleOps
+# People Ops
 
 ## Important Notes
 
-* **DO NOT share confidential HR information in the Mattermost pre-release server** \(a.k.a. [http://community.mattermost.com\](http://community.mattermost.com%29\) as it runs pre-production releases of Mattermost software and occasionally requires debugging of data stores. Use [BambooHR](https://mattermost.bamboohr.com/) to handle passport information, ID numbers and other sensitive data. HR issues \(e.g. health, extenuating circumstances, etc.\) should be discussed in @mattermost.com emails only.
+* **DO NOT share confidential HR information in the [Mattermost Community server](http://community.mattermost.com)**, as it runs pre-production releases of Mattermost software and occasionally requires debugging of data stores. Use [BambooHR](https://mattermost.bamboohr.com/) to handle passport information, ID numbers, and other sensitive data. HR issues \(e.g. health, extenuating circumstances, etc.\) should be discussed in @mattermost.com emails only.
```

Comment:
@natalie-hub, I changed the formatting because the previous formatting wasn't rendering properly. Please confirm that this is accurate. :)

Reply:
Thank you @justinegeffen - looks good :)